recipe/meta.yaml

```
@@ -43,9 +43,9 @@ requirements:
     - gitpython >=3.0.7,<4,!=3.1.19
     - pydeck >=0.8.0b4,<1
     - tornado >=6.0.3,<7
+    - typing-extensions >=4.3.0,<5
   run_constrained:
     - snowflake-snowpark-python >=1.17.0  # [py<312]
```

```suggestion
    - snowflake-snowpark-python >=1.17.0  # [py<312]
    - snowflake-connector-python >=2.8.0  # [py<312]
```
It is still set here: https://github.com/streamlit/streamlit/blob/1.41.1/lib/setup.py#L75